src/styles/index.scss

```diff
@@ -1,5 +1,12 @@
 @import url('https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css');
 
+// variables
```

Comment:
as we are going to increase this setup, it woudl be cool alreay to actually split this index.scss
into:
- one vendors.scss file (for importing bootstrap)
- one variables.scss file (for defining typically these $margin-medium)
- one other containing for now our new class definitions